cluster/addons/iso-client/coreaffinity/isolator.go

```diff
@@ -81,80 +80,99 @@ func getPod(bytePod []byte) (pod *v1.Pod, err error) {
 	return
 }
 
-// check wheter pod should be isolated
-func isIsoPod(pod *v1.Pod) bool {
-	if pod.Annotations["pod.alpha.kubernetes.io/isolation-api"] != "" {
-		return true
+func (i *isolator) gatherContainerRequest(container v1.Container) int64 {
+	resource, ok := container.Resources.Requests[v1.OpaqueIntResourceName(i.Name)]
+	if !ok {
+		return 0
 	}
-	return false
+	return resource.Value()
 }
 
-// extract isoSpec from annotations
-func getIsoSpec(annotations map[string]string) (spec *isoSpec, err error) {
-	spec = &isoSpec{}
-	err = json.Unmarshal([]byte(annotations["pod.alpha.kubernetes.io/isolation-api"]), spec)
-	if err != nil {
-		glog.Fatalf("Cannot unmarshal isoSpec: %v", err)
-		return nil, err
+func (i *isolator) countCoresFromOIR(pod *v1.Pod, cgroupInfo *lifecycle.CgroupInfo) int64 {
```

Comment:
Can we drop the `cgroupInfo` parameter? Doesn't appear to be used within the function body.